sections/components.tex

```diff
@@ -1,6 +1,8 @@
 % !TeX spellcheck = en_US
 \addsection{Components}{\skills/luck.png}
 
+\iftoggle{printable}{\vspace{-\baselineskip}}{}
```

Comment:
shouldn't this be part of \addsection?

Reply:
That can also simplify all the translations

Reply:
I wish it could be, but it's inconsistent, and this problem doesn't appear everywhere. Changing to `twoside` messes with the margins and seems to insert a newline randomly.

Reply:
And it doesn't impact translations at all, so no worries here.